libs/wingsdk/src/target-sim/secret.inflight.ts

```diff
@@ -11,18 +9,9 @@ import { Json, TraceType } from "../std";
 
 export class Secret implements ISecretClient, ISimulatorResourceInstance {
   private _context: ISimulatorContext | undefined;
-  private readonly secretsFile: string;
   private readonly name: string;
 
   constructor(props: SecretSchema) {
```

Comment:
Is it possible to have the secret values passed through `simulator.json` instead? I feel like the simulator's interface will be more robust if we can keep as many of the "simulation inputs" within the `target/app.wsim` folder that it gets passed as possible. Recall that to run the simulator, all you've needed to do is run this code more or less:

```js
let s = new sdk.Simulator({ simdir: "target/app.wsim" });
await s.start();
```

Reply:
https://github.com/winglang/wing/issues/6349

Reply:
> Is it possible to have the secret values passed through simulator.json instead?

I imagine its totally possible, but I think that it makes sense to suggest it in a different PR for 2 reasons. 

First at its heart this PR still needs to go in to fix the issue of [`.env` files not being loaded](https://github.com/winglang/wing/issues/6346) and thats really what this fixes. The code deleted in this file was just dead (which is why I removed it, we dont actually use that file to read values in the `values()` method

As well, I think it might require some discussion, as there may be complexity there still around how we decide what to write into the `simulator.json` I.E. the DX right now is that users define their secrets for the simulator in the `.env` files, and they cant really just place these values i the generated output dirs.

Reply:
Yea Im gonna start working on a quick PR for: https://github.com/winglang/wing/issues/6349 and maybe get a discussion going around the implementation.